RELEASE.md

```diff
@@ -15,28 +15,30 @@ RELEASE-TEMPLATE-BEGIN
 
 In parallel, the following releases can be done:
 
-* che-theia,
-* che-machine-exec,
-* che-devfile registry, then che-plugin-registry (once che-theia and machine-exec are done)
-* che-parent, then che-docs, then che as Release Candidate for QE
+* che-theia
+* che-machine-exec
+    * che-plugin-registry (once che-theia and machine-exec are done)
+* che-devfile registry
+* che-parent, then che-docs, then che (including [quay images](https://github.com/eclipse/che/blob/master/.ci/cico_common.sh#L123-L131)) to release-candidate branch
 
 Then in series:
 
-* che-parent, then che-docs, then che as Release
+* che-parent, then che-docs, then che (copy from release-candidate to release branch)
```

Comment:
I'm not sure I understand the difference between `che-parent, then che-docs, then che` above, followed by the same but in series.

Reply:
Ah nevermind, misread line 22.